docs/source/examples/best-practices.rst

```diff
@@ -0,0 +1,112 @@
+Best Practices
+==============
+
+
+Multi-GPUs
+~~~~~~~~~~
+
+When using multiple GPUs, if possible, it's best to co-locate as many GPUs on the same physical node.  This could be a
+DGX, a cloud instance with `multi-gpu options <https://rapids.ai/cloud>`_ , a high-density GPU HPC instance, etc.  This is done for
+two reasons:
+
+- 1. Moving data between GPUs is costly and performance decreases when computation stops due to communication overheads, Host-to-Device/Device-to-Host transfers, etc
+- 2. Mutli-GPU instances often come with accelerated networking like `NVLink <https://www.nvidia.com/en-us/data-center/nvlink/>`_.  These accelerated
+networking paths usually have much higher throughput/bandwidth compared with traditional networking *and* don't force and H-to-D/D-to-H transfers.  See `
```

Comment:
I would expand H-to-D/D-to-H to Host-to-Device/Device-to-Host. This will not be always immediately clear to a newcomer.